{
  "error.content_filtering.message": "The response was filtered due to the prompt triggering Azure OpenAIâ€™s content management policy. Please modify your prompt and retry.",
  "error.general_client.message": "Error happened during answering. Please check your internet connection and try again.",
  "error.general_server.message": "Sorry, we were unable to process your request at this time due to a server error. Please try again later. Thank you for your patience and understanding.",
  "error.429.message": "Due to high demand our AI capacities are overloaded. We understand the problem and continuously searching for the way to extend capacities of the service (unfortunately there limits on cloud providers). Please, try again in a couple of minutes or try another model",
  "error.401.message": "Authorization failed. Please reload the page and login again.",
  "error.403.message": "Forbidden",
  "error.400.message": "Invalid request",
  "error.unsupported_conversations_data_format.message": "Import of conversations failed because of unsupported data format",
  "error.unsupported_prompts_data_format.message": "Import of prompts failed because of unsupported data format",
  "error.local_storage_quota_exceeded.message": "Conversation storage capacity exceeded. Please clean up some conversations (prefer ones with media attachments) and try again.",
  "error.timeout_error.message": "Server is taking too long to respond due to either poor internet connection or excessive load. Please check your internet connection and try again. You also may try different model.",
  "error.custom_themes_config_not_provided.message": "The custom config host url not provided. Please recheck application settings",
  "error.error_getting_user_bucket.message": "Error happened during getting file user bucket. Please contact your administrator or try to reload the page.",
  "error.no_models_available.message": "You do not have any available models. Please contact your administrator or try to reload the page.",
  "error.import_conversations_failed.message": "Import of conversations failed",
  "error.import_prompts_failed.message": "Import of prompts failed",
  "error.export_failed.message": "Export failed",
  "error.share_failed.message": "Sharing failed. Please try again later.",
  "error.accept_share_failed.message": "Accepting sharing invite failed. Please open share link again to being able to see shared resource.",
  "error.accept_share_not_exists.message": "We are sorry, but the link you are trying to access has expired or does not exist.",
  "error.revoke_access_failed.message": "Revoking access failed. Please try again later.",
  "error.discard_shared_with_me_failed.message": "Discarding shared with you resource failed. Please try again later.",
  "error.share_by_me_listing_failed.message": "Getting shared by you resources failed. Please reload the page to get them again.",
  "error.share_with_me_listing_failed.message": "Getting shared with you resources failed. Please reload the page to get them again.",
  "error.not_valid_entity_type.message": "You made a request with an unavailable or nonexistent entity type",
  "error.publication_failed.message": "Publication failed. Please try again later.",
  "error.publications_upload_failed.message": "Publications uploading failed.",
  "error.publication_upload_failed.message": "Publication uploading failed.",
  "error.publication_deletion_failed.message": "Publication deletion failed.",
  "error.published_items_upload_failed.message": "Published items uploading failed.",
  "error.publication_approve_failed.message": "Publication approving failed.",
  "error.publication_reject_failed.message": "Publication rejecting failed.",
  "error.publishing_by_me_items_uploading_failed.message": "Published by me items uploading failed.",
  "error.rules_uploading_failed.message": "Rules uploading failed."
}
